kv.go

```diff
@@ -590,6 +596,7 @@ func (kv *kvs) History(key string, opts ...WatchOpt) ([]KeyValueEntry, error) {
 type watcher struct {
 	updates chan KeyValueEntry
 	sub     *Subscription
+	closed  uint32
```

Comment:
When using with atomic ops, it is preferable to move those at the top of the struct. If people don't know and later add, say a boolean, between sub and closed, it will panic on a 32bit release.

See https://github.com/golang/go/issues/599

Reply:
Will do.